src/components/Downloads/v2index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext, useMemo } from 'react';
+
+import EMGTable from 'components/UI/EMGTable';
+import { MGnifyDatum } from 'hooks/data/useData';
+import AnalysisContext from 'pages/Analysis/V2AnalysisContext';
+
+const Downloads: React.FC = () => {
+  const { overviewData } = useContext(AnalysisContext);
+  const downloads = overviewData?.downloads;
```

Comment:
This will come from a separate endpoint in V2. `analyses/${accession}/downloads`. (Not yet present on API v2 surface)